### PR TITLE
First Task

### PR DESCRIPTION
**Pull request by Nursultan91**
Первое задание из квеста RoR для мелких

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/*!
+ * Bootstrap v3.3.7 (http://getbootstrap.com)
```
Comment (X1ting) [inline app/assets/stylesheets/application.scss]: what is it?

File: app/assets/javascripts/pages.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (X1ting) [inline app/assets/javascripts/pages.coffee]: are you need this file?

File: app/assets/stylesheets/pages.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Pages controller here.
```
Comment (X1ting) [inline app/assets/stylesheets/pages.scss]: are you need this file?

File: app/helpers/pages_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
```
Comment (X1ting) [inline app/helpers/pages_helper.rb]: are you need this?

File: app/views/layouts/_navbar.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<!-- Static navbar -->
+    <nav class="navbar navbar-default navbar-static-top">
```
Comment (X1ting) [inline app/views/layouts/_navbar.html.erb]: indents

File: app/views/pages/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<section id="container">
+
```
Comment (X1ting) [inline app/views/pages/index.html.erb]: indents

File: test/models/card_test.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CardTest < ActiveSupport::TestCase
```
Comment (X1ting) [inline test/models/card_test.rb]: if you add test file, then wrote tests, otherwise delete this.

File: app/views/layouts/_navbar.html.erb [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+      <div class="container">
+        <div class="navbar-header">
+          <a class="navbar-brand" href="#">Флэшкарточкер</a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="#">Все карточки</a></li>
```
Comment (X1ting) [inline app/views/layouts/_navbar.html.erb]: it is right link?

File: app/assets/stylesheets/application.css [hunk 9]
```diff
@@ -1,15 +0,0 @@
-/*
```
Comment (X1ting) [inline app/assets/stylesheets/application.css]: Which manifest file you want to use instead of this?

File: Gemfile [hunk 10]
```diff
@@ -4,7 +4,8 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 gem 'rails', '~> 5.1.4'
-gem 'sqlite3'
+# gem 'sqlite3'
```
Comment (X1ting) [inline Gemfile]: why you not delete this? Afraid to forgot gem name?)

File: app/assets/javascripts/pages.coffee (hunk 2 above)
Comment (Nursultan91) [reply app/assets/javascripts/pages.coffee]: Hey dude, where did you learn english? as i remember when you use present simple it looks like "Do you need this file?"

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (Nursultan91) [reply app/assets/stylesheets/application.scss]: i have no idea

File: app/views/layouts/_navbar.html.erb (hunk 5 above)
Comment (Nursultan91) [reply app/views/layouts/_navbar.html.erb]: podrobnee

File: app/views/layouts/_navbar.html.erb (hunk 8 above)
Comment (Nursultan91) [reply app/views/layouts/_navbar.html.erb]: no

File: app/assets/stylesheets/application.scss [hunk 11]
```diff
@@ -0,0 +1,2 @@
...last-child>.btn,.input-group-btn:last-child>.btn-group{z-index:2;margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:focus,.nav>li>a:hover{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:focus,.nav>li.disabled>a:hover{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:focus,.nav .open>a:hover{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:focus,.nav-tabs>li.active>a:hover{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-j
```
Comment (X1ting) [inline app/assets/stylesheets/application.scss]: what is it?

File: app/assets/stylesheets/application.css (hunk 9 above)
Comment (X1ting) [reply app/assets/stylesheets/application.css]: ?

File: app/views/pages/index.html.erb [hunk 12]
```diff
@@ -0,0 +1,3 @@
+<p style="font-size: 22px;">
+Первый в мире удобный менеджер флеш-карточек. Именно так.
```
Comment (X1ting) [inline app/views/pages/index.html.erb]: indent

File: app/views/pages/index.html.erb [hunk 13]
```diff
@@ -0,0 +1,3 @@
+<p style="font-size: 22px;">
```
Comment (X1ting) [inline app/views/pages/index.html.erb]: move it to style

File: test/models/card_test.rb (hunk 7 above)
Comment (X1ting) [reply test/models/card_test.rb]: !

Comment (Nursultan91) [reply test/models/card_test.rb]: How can i write test? I have never done it

File: app/assets/stylesheets/application.css (hunk 9 above)
Comment (Nursultan91) [reply app/assets/stylesheets/application.css]: applocation.scss

File: app/assets/stylesheets/application.scss (hunk 11 above)
Comment (Nursultan91) [reply app/assets/stylesheets/application.scss]: bootstrap

File: app/views/pages/index.html.erb (hunk 13 above)
Comment (Nursultan91) [reply app/views/pages/index.html.erb]: it was in first task and i were must to create tha same file as it was in archive

File: test/models/card_test.rb (hunk 7 above)
Comment (Nursultan91) [reply test/models/card_test.rb]: deleted

File: app/views/pages/index.html.erb (hunk 12 above)
Comment (Nursultan91) [reply app/views/pages/index.html.erb]: done

**Pull request merged**